src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -375,19 +338,27 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
                                 break;
 
                             case "?":
-                                await ShowDialog(message, allowEval, MessageButtons.YesNoCancel, CancellationTokenSource.CreateLinkedTokenSource(ct, _cancelAllCts.Token).Token);
+                                ShowDialog(message, MessageButtons.YesNoCancel, CancellationTokenSource.CreateLinkedTokenSource(ct, _cancelAllCts.Token).Token)
+                                    .SilenceException<MessageTransportException>()
+                                    .DoNotWait();
                                 break;
 
                             case "??":
-                                await ShowDialog(message, allowEval, MessageButtons.YesNo, CancellationTokenSource.CreateLinkedTokenSource(ct, _cancelAllCts.Token).Token);
+                                ShowDialog(message, MessageButtons.YesNo, CancellationTokenSource.CreateLinkedTokenSource(ct, _cancelAllCts.Token).Token)
```

Comment:
`ShowDialog` has 2 awaits - `await _callbacks.ShowDialog(contexts, s, _canEval, buttons, ct);` and `await RespondAsync(request, ct, response);` Do we want to apply `DoNotWait` for both?